CMake/crosscompile-arch-config.cmake

```
@@ -56,7 +56,7 @@ elseif(BOARD MATCHES "RPI3" OR BOARD MATCHES "CORTEXA53")
   set(OPENBLAS_TARGET "CORTEXA53")
   set(OPENBLAS_BINARY "64")
 elseif(BOARD MATCHES "RPI4" OR BOARD MATCHES "CORTEXA72")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mtune=cortex-a72 -ftree-vectorize")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=armv8.2-a+crypto+fp16+rcpc+dotprod -asynchronous-unwind-tables")
```

I think this is what I used for the RPi5, not the RPi4---did you find that it worked there too?

---

```suggestion
  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=armv8.2-a+crypto+fp16+rcpc+dotprod -fasynchronous-unwind-tables")
```
Also this will fix the build issue... took a while to dig to the bottom of that.  Opening a PR that will make it easier to find the issue in just a moment...